compiler/noirc_evaluator/src/ssa/ir/function.rs

```diff
@@ -174,8 +174,7 @@ impl Function {
                 break;
             }
         }
-        function_return_values
-            .expect("Expected a return instruction, as function construction is finished")
+        function_return_values.unwrap_or(&Vec::new()).clone()
```

Comment:
Can you also update this function to directly return instead of using `function_return_values` then breaking? If we reach after the loop we could then return `&[]` without needing an expect or anything.

Reply:
Done